### DISCUSSION TITLE
RSS feed for my home timeline

### DISCUSSION THREAD

**Discussion by jose-dvm** [mastodon/mastodon#18432] [Q&A] [answered]
Hi,

I know there is an RSS feed available for posts from an specific account.

The URL looks like https://instance/users/user.rss

But there is a feed for my home's timeline where I can find posts from the users I follow?

Something like Github's private news feed.

It would be great to follow accounts from an RSS reader using a unique feed when using online services with feeds limitations.

**Comment by ClearlyClaire** [comment, marked as answer]
No, there is currently no such feature.

**Reply by jose-dvm** [reply]
Aah, I see.

Thanks for the answer, Claire.

**Comment by lprimak** [comment]
Ok, I'd like to request such a feature. thank you!

**Reply by afontenot** [reply]
There is already a feature request for this here: https://github.com/mastodon/mastodon/issues/18601

**Reply by powerbiferrytales** [reply]
There is the option to view all the posts of everyone with whom you have interacted. As your RSS feed 'with_replies.rss' includes their username, you can then use their username as a parameter and retrieve all of their posts. So not everyone you follow, just everyone whio has interacted with you. Close enough maybe?

**Reply by lprimak** [reply]
Thanks! That does help, but unfortunately not enough :(

**Reply by powerbiferrytales** [reply]
So close...so far! 😄
